microservices/scraper-thaiNGO/app.py

```diff
@@ -0,0 +1,14 @@
+from flask import Flask
+from scraper import basepage_scrape
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def my_first_route():
+    basepage_scrape()
+    return str(basepage_scrape())
+
```

Comment:
Can we also add another route that returns the list if routes available for this scraper